doc/enterprise/overview.md

```diff
@@ -0,0 +1,30 @@
+# Overview
+
+![alt tag](enterprise.png)
+
+Pachyderm Enterprise Edition includes everything you need to scale and manage Pachyderm data pipelines in an enterprise setting. It is secure and simple, and it delivers the most recent version of Pachyderm along with:
+
+- Administrative and security features needed for enterprise-scale implementations of Pachyderm 
+- Visual and interactive interfaces to Pachyderm  
+
+Pachdyerm Enterprise Edition can be deployed easily on top of an existing or new deployment of Pachyderm, and we have engineers available to help enterprise customers get up and running very quickly.  To get more information about Pachyderm Enterprise Edition, to ask questions, or to get access for evaluation, please contact us at [sales@pachyderm.io](mailto:sales@pachyderm.io) or on our [Slack](http://slack.pachyderm.io/). 
+
+## Pipeline Visualization and Data Exploration
+
+![alt tag](dashboard.png)
+
+Pachyderm Enterprise Edition includes a full UI, or dashboard, for visualizing your pipelines and exploring your data.  Pachyderm Enterprise will automatically infer the structure of your Pachyderm DAGs and display them to you visually.  You can even click on individual segments of the pipelines to see what data is being processed, how many jobs have run, what images and commands are being run in your pipeline stages, and much more!  You can also explore the versioned data in your Pachyderm data respositories and see the commit history of your data.  
```

Comment:
the tone of this paragraph is different than the previous. it's all "you can even" or "you can also". Make it a littler more enterprise-y. Change the you's to Data Scientists or cluster admins, etc.